profile/views.py

```diff
@@ -29,6 +30,17 @@ def profile(request):
     return render(request, 'user_profile/profile.html', context)
 
 
+@login_required
+def delete_account(request):
+    """Delete account view"""
+    if request.method == 'POST':
+        user = request.user
+        logout(request)  # Log out the user before deleting the account
+        user.delete()  # Delete the user account
```

Comment:
Will it delete all created by user pages?

Reply:
![image](https://github.com/Strexas/qr_web/assets/83160542/09859939-dec8-44c9-8452-99fbbf413ed5)

Reply:
Yes